src/clr-angular/utils/drag-and-drop/interfaces/draggable-event.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {DragPosition} from "./client-position";
+import {Draggable} from "./draggable";
+
+export interface DraggableEvent {
```

Comment:
Same thing that we mentioned above. Interfaces typically need comments because we dont have any implementation to understand what each property does.

Reply:
Will address this issue.